src/util.cpp

```diff
@@ -364,7 +362,7 @@ void ParseParameters(int argc, const char* const argv[])
         }
 #ifdef WIN32
         boost::to_lower(str);
-        if (boost::algorithm::starts_with(str, "/"))
+        if (str.size() > 0 && str[0] == '/')
```

Comment:
The algorithm predicate way of writing things is much more modern C++.
If we'd do this, we should just provide our own implementation of `boost::algorithm::starts_with`.
But I don't think replacing that is urgent.